lib/usage.js

```diff
@@ -397,8 +397,11 @@ module.exports = function (yargs, y18n) {
 
   // guess the width of the console window, max-width 80.
   function windowWidth () {
-    const wsize = require('window-size')
-    return wsize.width ? Math.min(80, wsize.width) : null
+    if (process && process.stdout && process.stdout.columns && Math.min(80, process.stdout.colums)) {
```

Comment:
I agree, this should probably look something more like this:

```
if (typeof process === 'object' && process.stdout && process.stdout.columns) {
  return Math.min(80, process.stdout.colums)
} else {
  return 80
}
```

I think it's probably more correct to explicitly return `80` than null, this might be a historical bug.